dist/common/scripts/scylla_raid_setup

```diff
@@ -131,7 +131,47 @@ if __name__ == '__main__':
             f.write(res)
 
     makedirs(mount_at)
-    run('mount -t xfs -o noatime {raid} "{mount_at}"'.format(raid=fsdev, mount_at=mount_at))
+    mntunit_bn = out('systemd-escape -p --suffix=mount {}'.format(mount_at))
+    mntunit = Path('/etc/systemd/system/{}'.format(mntunit_bn))
+    if mntunit.exists():
+        print('mount unit {} already exists'.format(mntunit))
+        sys.exit(1)
+
+    res = out('blkid {}'.format(fsdev))
+    match = re.search(r'^/dev/\S+: (UUID="\S+")', res.strip())
+    uuid = match.group(1)
+    unit_data = f'''
+[Unit]
+Description=Scylla data directory
+Conflicts=umount.target
```

Comment:
The manual says:

       ·   All mount units acquire automatic Before= and Conflicts= on umount.target in order to be stopped during shutdown.

So this looks unneeded.